elevenlabs_mcp/server.py

```
@@ -55,9 +64,45 @@
 mcp = FastMCP("ElevenLabs")
 
 
+@mcp.resource("elevenlabs://{filename}")
+def get_elevenlabs_resource(filename: str) -> Resource:
+    """
+    Resource handler for ElevenLabs generated files.
+    """
+    # Construct the file path
+    output_path = make_output_path(None, base_path)
+    file_path = output_path / filename
+
+    if not file_path.exists():
+        raise FileNotFoundError(f"Resource file not found: {filename}")
+
+    # Read the file and determine MIME type
+    with open(file_path, "rb") as f:
+        file_data = f.read()
+
+    file_extension = file_path.suffix.lstrip(".")
+    mime_type = get_mime_type(file_extension)
+
+    # For text files, return text content
+    if mime_type.startswith("text/"):
+        try:
+            text_content = file_data.decode("utf-8")
+            return Resource(
+                uri=f"elevenlabs://{filename}", mimeType=mime_type, text=text_content
+            )
+        except UnicodeDecodeError:
+            pass
```

Is the idea that we fallback to binary encoding if this fails? I think that might be unexpected, might be better to throw an error here since that probably means that the mime type is incorrect or the contents are corrupt.